protobuf/flow/execution/execution.proto

```diff
@@ -105,6 +105,7 @@ message GetEventsForBlockIDsResponse {
     repeated entities.Event events = 3;
   }
   repeated Result results = 1;
+  bool is_ccf = 2;
```

Comment:
does it make sense to instead use a more generic: `eventEncodingVersion`

Reply:
Great suggestion Janez!  👍 

It would be helpful if `eventEncodingVersion` identifies the combination of encoding format + format version. E.g. if it's an enum then members like:
- JSON_CDC_V0
- ...
- CCF_V0

Reply:
Done!